### PR TITLE
Reorganize the repo for more KDFs

### PR DESCRIPTION
**Pull request by vladikoff**
~~Blocked by https://github.com/RustCrypto/KDFs/issues/13~~

### CODE REVIEW

File: hkdf/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[package]
+name = "hkdf"
+version = "0.4.1-alpha.0"
+authors = ["vladikoff", "warner"]
+license = "MIT"
```
Comment (vladikoff) [inline hkdf/Cargo.toml]: need to change this for dual license

Comment (newpavlov) [reply hkdf/Cargo.toml]: And also add `LICENSE-APACHE` file.

Comment (vladikoff) [reply hkdf/Cargo.toml]: Yeap did that :)

File: hkdf/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# rust-hkdf
+
+[![Build Status](https://travis-ci.org/vladikoff/rust-hkdf.svg?branch=master)](https://travis-ci.org/vladikoff/rust-hkdf)
```
Comment (vladikoff) [inline hkdf/README.md]: need to update the badge

File: hkdf/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+[dependencies]
+hkdf = "*"
+```
+
+From the git repository:
+
+```toml
+[dependencies.hkdf]
+
+git = "https://github.com/vladikoff/rust-hkdf.git"
```
Comment (vladikoff) [inline hkdf/README.md]: need to update this here

File: .travis.yml [hunk 4]
```diff
@@ -1,10 +1,35 @@
...
+    - env: TARGET=x86_64-unknown-linux-gnu
+      rust: 1.21.0
+    - env: TARGET=x86_64-unknown-linux-gnu
+      rust: nightly
+    - env: TARGET=powerpc-unknown-linux-gnu
+      rust: stable
+    - env: TARGET=powerpc64-unknown-linux-gnu
+      rust: stable
+    # tests if crates truly can be built without std
+    - env: TARGET=thumbv7em-none-eabi
```
Comment (newpavlov) [inline .travis.yml]: This test should be removed, as currently `hkdf` can not be used in no_std environments.

**Comment by vladikoff** [comment]
@newpavlov r?

**Pull request merged**